mindjunkies/courses/models.py

```
@@ -138,6 +140,7 @@
 
 
 class Enrollment(BaseModel):
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)